src/main/java/org/jenkinsci/plugins/DependencyTrack/JobAction.java

```diff
@@ -26,9 +26,9 @@
 
 public class JobAction implements Action {
 
-    private AbstractProject<?, ?> project;
+    private Job<?, ?> project;
```

Comment:
AbstractProject is a subclass of Job, but some things want Job, so use superclass.